text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/tag/standard/StandardTags.java

```diff
@@ -196,6 +198,16 @@ public static TagResolver reset() {
     return RESET;
   }
 
+  /**
+   * Get a resolver for the {@value NewLineTag#NEW_LINE} tag.
+   *
+   * @return a resolver for the {@value NewLineTag#NEW_LINE} tag.
```

Comment:
`@value` could be changed to `@link` here probably

Reply:
`@value` gives a link to the constant as well

Reply:
Oh, I've never used this tag so I remembered how it worked wrongly. I read up on it now.

Reply:
It still has to be `NewlineTag` instead of `NewLineTag` though so I'll leave this unresolved

Reply:
I'll fix that in the next commit.